Prevent Segfault by checking for existance in HashMap before adding

richardeigenmann opened the pull request.

richardeigenmann committed:
Prevent Segfault by checking for existance in HashMap before adding

qris commented:
Thanks Richard for fixing this!

Sent from my iPhone

> On 14 Feb 2016, at 23:16, Richard Eigenmann notifications@github.com wrote:
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/boxbackup/boxi/pull/11
> 
> Commit Summary
> 
> Prevent Segfault by checking for existance in HashMap before adding
> File Changes
> 
> M src/RestoreFilesPanel.cc (8)
> Patch Links:
> 
> https://github.com/boxbackup/boxi/pull/11.patch
> https://github.com/boxbackup/boxi/pull/11.diff
> —
> Reply to this email directly or view it on GitHub.

richardeigenmann commented:
I fixed the segfault. I'm not so sure about any 'feature' I introduced...
But I figure we are better off with features in the app than it being
broken. ☺ At least now I can actually use it!
On 15 Feb 2016 18:06, "Chris Wilson" notifications@github.com wrote:

> Thanks Richard for fixing this!
> 
> Sent from my iPhone
> 
> > On 14 Feb 2016, at 23:16, Richard Eigenmann notifications@github.com
> > wrote:
> > 
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/boxbackup/boxi/pull/11
> > 
> > Commit Summary
> > 
> > Prevent Segfault by checking for existance in HashMap before adding
> > File Changes
> > 
> > M src/RestoreFilesPanel.cc (8)
> > Patch Links:
> > 
> > https://github.com/boxbackup/boxi/pull/11.patch
> > https://github.com/boxbackup/boxi/pull/11.diff
> > —
> > Reply to this email directly or view it on GitHub.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/boxbackup/boxi/pull/11#issuecomment-184303899.
